xrpl/common/credential_test.go

```diff
@@ -0,0 +1,50 @@
+package common
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/Peersyst/xrpl-go/xrpl/transaction/types"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestIsValidCredentialType(t *testing.T) {
+	tests := []struct {
+		name     string
+		credType types.CredentialType
+		expected bool
+	}{
+		{
+			name:     "empty string",
```

Comment:
Add `pass` and `fail` preffixes

Reply:
ah right, forgot to add them here. Thanks for the catch